inspirehep/modules/search/query_factory.py

```diff
@@ -65,18 +67,10 @@ def invenio_query(pattern):
         try:
             search_walker = ElasticSearchNoKeywordsDSL()
             query.accept(search_walker)
-            query = Q('multi_match', query=pattern, fields=[
-                "title^3",
-                "title.raw^10",
-                "abstract^2",
-                "abstract.raw^4",
-                "author^10",
-                "author.raw^15",
-                "reportnumber^10",
-                "eprint^10",
-                "doi^10"],
-                zero_terms_query="all"
-            )
+            query = Q('multi_match',
+                      query=pattern,
+                      fields=DEFAULT_FIELDS_BOOSTING[index],
```

Comment:
Might not be a problem with https://github.com/inspirehep/inspire-next/pull/1098/files#r63504108